docs/sources/logql/functions.md

```diff
@@ -0,0 +1,370 @@
+---
+title: Template functions
```

Comment:
If you care about the order of the topics in the side menu, please add a weight in the frontmatter.